templates/aws-stack.yml

```
@@ -127,13 +127,13 @@ Parameters:
     Default: "stable"
 
   BuildkiteAgentToken:
-    Description: Buildkite agent registration token. Deprecated, use BuildkiteAgentTokenParameterStorePath instead.
+    Description: Buildkite agent registration token. Or, preload it into SSM Parameter Store and use BuildkiteAgentTokenParameterStorePath for secure environments.
```

i think we could probably use some slightly stronger wording here - maybe something like 

```suggestion
    Description: Buildkite agent registration token. For production workloads, we recommend preloading the token into SSM Parameter Store and using BuildkiteAgentTokenParameterStorePath instead.
```

i think we want to make it clear that the SSM is the golden path for prod workloads, and minimise confusion between the two parameters.

---

I'm not sure that we're sure that we SSM is the golden path.
I think step one is making it clear that both work, and step two (out of scope for this PR) is to lay a roadmap for consolidating them… possibly even to _neither_ of the existing options. e.g. Secrets Manager.

---

right! makes sense. in that case, LGTM :)